static/scripts/rewards/cirip/query-reverse-ens.ts

```diff
@@ -2,6 +2,8 @@ import { app } from "../app-state";
 import { useRpcHandler } from "../web3/use-rpc-handler";
 import { ethers } from "ethers";
 
+const mainnetRpcUrl = "https://eth.api.onfinality.io/public";
```

Comment:
Maybe you can hit the "." key and edit the file here on GitHub. It might be faster.

Reply:
It will require a fresh pull as I'd need to update the handler package, settings etc and verify it's working.

I just tried it locally and it's throwing an error when running `git submodule update` and pointing to `permit-generation` so that needs updated and verified too.

Both introduce breaking changes so I'll probably be better off with a separate PR all together as opposed to PRing against this.

---

Allow a top RPC from `chainlist.org` to be hardcoded and I will update it in my PR?

Reply:
So then let's resolve and you can approve this merge

Reply:
It's difficult for me to test because I don't have an ENS myself. this endpoint while consistently in the lower half of those on `chainlist` I haven't seen it drop off while watching for 5-10 mins.

It should be okay until I catch it tonight